androidApp/home/src/main/java/tech/takahana/iconwallpaper/android/home/ui/components/ImagePattern.kt

```diff
@@ -0,0 +1,40 @@
+package tech.takahana.iconwallpaper.android.home.ui.components
+
+import androidx.compose.foundation.Canvas
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Surface
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.ImageBitmap
+import androidx.compose.ui.res.imageResource
+import androidx.compose.ui.tooling.preview.Preview
+import tech.takahana.iconwallpaper.android.core.ui.theme.IconWallPaperTheme
+import tech.takahana.iconwallpaper.android.home.R
+import tech.takahana.iconwallpaper.android.home.ui.util.DrawScopeUtils.drawPattern
+import tech.takahana.iconwallpaper.shared.domain.domainobject.PatternType
+
+@Composable
+fun ImagePattern(patternType: PatternType, resourceId: Int) {
```

Comment:
Idを引数に取るときはアノテーションをつけるのが良いです！
そうするとIDEで警告が出たりして、意図したいIntの値を入れるのを防げたりします。

```suggestion
fun ImagePattern(patternType: PatternType, @DrawableRes resourceId: Int) {
```

Reply:
こちらのコミットで修正しました！

117b9d3f062e6f02277400f5259929fb99836519